docs/sdk/1-workshop/6-verification.md

```diff
@@ -4,68 +4,58 @@ title: 🤝 Verification
 ---
 
 import CodeBlock from '@theme/CodeBlock';
-import VerifierIndex from '!!raw-loader!../../../code-examples/workshop/verifier/index.js';
+import Verify from '!!raw-loader!../../../code-examples/workshop/verify.js';
 import CreatePresentation from '!!raw-loader!../../../code-examples/workshop/claimer/createPresentation.js';
-import ClaimerIndex from '!!raw-loader!../../../code-examples/workshop/claimer/index.js';
-import Index from '!!raw-loader!../../../code-examples/workshop/index.js';
 
 In this section, you'll play the role of a <span class="label-role verifier">Verifier</span>:
 
-- You'll take an `Credential` object given to you by a <span class="label-role claimer">Claimer</span>;
+- You'll take an `Presentation` object given to you by a <span class="label-role claimer">Claimer</span>;
 - You'll verify that its data is correct;
 - You'll verify that the attestation is valid, i.e. its hash exists on-chain and the attestation has not been revoked.
-- You'll verify that the `Credential` is owned by the  <span class="label-role claimer">Claimer</span> sending it
+- You'll verify that the `Credential` is owned by the <span class="label-role claimer">Claimer</span> sending it
 
 :::info Credential
 
-A `Credential` object is also called an Attested Claim: It is what <span class="label-role claimer">claimers</span> present to <span class="label-role verifier">verifiers</span> upon request.
+A `Credential` object is also called an Attested Claim: It is what <span class="label-role claimer">claimers</span> need to create `Presentations`.
 
 :::
 
-## Verifier Index
+:::info Presentation
 
-Let's create our `verifier/index.js`. Here we'll expose `getChallenge` which returns a random and unique
-challenge for the  <span class="label-role claimer">Claimer</span> to sign, this is used to prove ownership. We'll also expose `verifyCredential`
-which will do the actual verification. Copy the code below, this completes the  <span class="label-role verifier">Verifier</span> code!
+A `Presentation` object is created by the <span class="label-role claimer">claimers</span> using a credential.
```

Comment:
This doesn't makes sense. The claimer creates a credential from what?

Reply:
But that assumes the user knows what a presentation is and what they make it from. Being explicit is much better

Reply:
```suggestion
A credential creates the `Presentation` object by the <span class="label-role claimer">claimers</span>.
```

Reply:
The concept is fine, but I mean the way it reads.